src/types/DataGenerator.cpp

```diff
@@ -319,6 +328,146 @@ void DataGenerator::saveModel(std::string fileName, float lambda,
   }
 }
 
+std::tuple<PerformanceTable, std::unordered_map<std::string, Category>>
+DataGenerator::loadDataset(std::string fileName) {
```

Comment:
Should we return an AlternativesPerformance here instead of the tuple ?

Reply:
yup changed